CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 #### Features
 
+* [#1463](https://github.com/ruby-grape/grape/pull/1463): Fix array indicies in error messages - [@ffloyd](https://github.com/ffloyd).
```

Comment:
@dblock As you can see, my english language skill is far from perfect. I've notices that some PRs uses "[verb]s" form  #1153, some just use "[verb]" #1378. So, maybe both variants are correct?